pkgs/tools/misc/arxiv-latex-cleaner/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, python3Packages
+, fetchFromGitHub
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "google-research";
```

This should probably match the name in the PR title.
```suggestion
  pname = "arxiv-latex-cleaner";
```

---

Oops. sorry about the mixup. fixed.